concurrent/src/test/java/io/airlift/concurrent/TestMoreFutures.java

```diff
@@ -815,7 +810,7 @@ private static void assertFailure(Thrower thrower, Consumer<Throwable> verifier)
             verifier.accept(throwable);
             return;
         }
-        fail("expected exception to be thrown");
+        abort("expected exception to be thrown");
```

Comment:
this was a failure. is `abort` a failure or "skip exception"?

(same in other places)